cors/handler_test.go

```diff
@@ -0,0 +1,89 @@
+package cors_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
+
+	"github.com/SKF/go-enlight-middleware/cors"
+)
+
+func Test_Get(t *testing.T) {
```

Comment:
I propose we add what we are testing into the name. Here you are asserting that the AccessControlAllowOrigin header is set on all requests, so that should probably be more clear from the name